lib/maths/CDataFrameCategoryEncoder.cc

```diff
@@ -325,6 +325,26 @@ bool CDataFrameCategoryEncoder::restore(core::CStateRestoreTraverser& traverser,
     return true;
 }
 
+const CDataFrameCategoryEncoder::TEncodingUPtrVec& CDataFrameCategoryEncoder::encodings() const {
+    return m_Encodings;
+}
+
+void CDataFrameCategoryEncoder::accept(CDataFrameCategoryEncoder::Visitor& visitor) {
```

Comment:
I'd be inclined to make this const
```suggestion
void CDataFrameCategoryEncoder::accept(CDataFrameCategoryEncoder::Visitor& visitor) const {
```

Reply:
Done